app/api/exports.py

```diff
@@ -264,6 +275,7 @@ def export_speakers_csv(event_identifier):
 
 @export_routes.route('/events/<string:event_identifier>/export/sessions/pdf', methods=['GET'],
                      endpoint='export_sessions_pdf')
+@get_event_id
 @is_coorganizer
 def export_sessions_pdf(event_identifier):
```

Comment:
Now this should be event_id and conversion in each function should be removed

Reply:
I will edit the logic accordingly.